app/views/editor/_group_by_subfield.html.erb

```diff
@@ -22,11 +22,12 @@
 <%- end %>
 </dl>
 
+<% unless @item.record_type == 11 %>
```

Comment:
Can we think about another way to do this? Maybe with something more explicit in the configuration?

Reply:
Yes, I am not glad with this too. But otherwise the complex structure of the configuration will be blowed up again by another element. Anyway: Proposals are welcome..

Reply:
I have made an issue with that and removed the lines with the recent commit, so we have time to find a better solution